exotations/task_maps/task_map/init/Point2Line.in

```diff
@@ -7,3 +7,4 @@ extend <exotica/TaskMap>
 
 Required Eigen::Vector3d EndPoint;  // point in 'Base' frame that defines the end of the line segment
 Optional bool infinite = true;      // true: interpret 'EndPoint' as the direction of an infinite line
```

Comment:
I just noticed all the other Initialisers start with capital letters for parameters, can we make this one the same? 

If we get this rebased then on master and a few of the erroneous commits squashed I think it's ready for merging :-)